### ISSUE TITLE
Public refresh

### ISSUE THREAD

**Issue by caalle** [4degrees/riffle#10]
Would be good to have a public method to refresh the items in the browser. If I have a reference of the browser and once opened the available items will not be updated.

```
browser = riffle.browser.FilesystemBrowser(parent=self)
browser.refresh()

# To be able to specify if the current location should be persisted would be nice as well.
browser.refresh(persist_location=True)
```

**Comment by martinpengellyphillips** [comment]
Makes sense. The other option would be to expose the model more and handle the refresh there (which would subsequently update the view). A convenience method at the higher level would still be useful though I think. PR welcome.
